components/layout/SharePopup/styles.js

```diff
@@ -1,8 +1,20 @@
 import styled from "styled-components";
+import { Popover } from "@headlessui/react";
 
 export const PanelInner = styled.div`
   padding: 0.5em;
   background-color: var(--white);
   border-radius: 0.5em;
   box-shadow: 5px 15px 35px 8px rgba(0, 0, 0, 0.13);
 `;
+
+export const Button = styled(Popover.Button)`
```

Comment:
A small unrelated fix to the lil share menu: it didn't have any visible hover/focus/open styling.  Decided a simple outline felt the most appropriate and flexible to how many background colors this button gets used over.